tests/models/distilbert/test_modeling_distilbert.py

```diff
@@ -285,6 +287,112 @@ def test_torchscript_device_change(self):
                 loaded = torch.jit.load(os.path.join(tmp, "traced_model.pt"), map_location=torch_device)
                 loaded(inputs_dict["input_ids"].to(torch_device), inputs_dict["attention_mask"].to(torch_device))
 
+    @require_flash_attn
+    @require_torch_gpu
+    @mark.flash_attn_test
+    @slow
+    def test_flash_attn_2_inference(self):
+        import torch
+
+        for model_class in self.all_model_classes:
+            dummy_input = torch.LongTensor(
+                [
+                    [1, 2, 3, 4],
```

Comment:
Because `DistilBertForMultipleChoice` requires inputs with different shapes we need to override this test.

Reply:
Could you add this as a comment at the top of the test?